code/Model/Submission/SubmittedForm.php

```diff
@@ -39,8 +39,8 @@ class SubmittedForm extends DataObject
     ];
 
     private static $summary_fields = [
-        'ID',
-        'Created'
+        'ID' => 'ID',
```

Comment:
```suggestion
        'ID'
```
You don't need to make this associative unless you're including a label that differs from the field name.